modules/oadp-self-service-admin-spec-enforce-nab.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service-admin-use-cases.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-self-service-admin-spec-enforce-nab_{context}"]
+= Self-Service administrator spec enforcement for NAB
+
+As a cluster administrator, you can enforce the following fields for a `NonAdminBackup` (NAB) CR:
+
+* `csiSnapshotTimeout`
+* `itemOperationTimeout`
+* `resourcePolicy`
+* `includedResources`
+* `excludedResources`
+* `orderedResources`
+* `includeClusterResources`
+* `excludedClusterScopedResources`
+* `excludedNamespaceScopedResources`
+* `includedNamespaceScopedResources`
+* `labelSelector`
+* `orLabelSelectors`
+* `snapshotVolumes`
+* `ttl`
+* `snapshotMoveData`
+* `uploaderConfig.parallelFilesUpload`
+
+For example, if you want to enforce a `ttl` value of a backup, and want to enforce Data Mover for a non-admin user, you can set up the `DataProtectionApplication` (DPA) CR as following:
+
+.Example `DataProtectionApplication` CR
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+...
+spec:
+  nonAdmin:
+    enable: true
+    enforceBackupSpec: # <1>
+      snapshotMoveData: true # <2>
+      ttl: 158h0m0s # <3>
+----
+
+In the example DPA, you have:
+
+* Added the `enforceBackupSpec` section as shown in annotation *1*.
+* Enforced the Data Mover by setting the `snapshotMoveData` field to `true`, as shown in annotation *2*. 
+* Enforced the `ttl` value by setting the field to 158h0m0s, as shown in annotation *3*.
+
+When a non-admin user creates a NAB, they must follow the template set up in the DPA, otherwise, they get an error.
```

Comment:
I would replace "error" w/ phase "backing off"  @mpryc FYI.   
I would replace error on all the lines referring to bsl or b/r enforcement :)

Reply:
phase: BackingOff